app/views/index/index.html.erb

```diff
@@ -8,7 +8,7 @@
 </div>
 
 <div class="row justify-content-center">
-  <div class="col-xs-12 col-sm-11 col-md-6 mt-5">
```

Comment:
Why did you delete this class?

Reply:
@gfedorenko i remove only col-md-6 for remove between break